e2e-manifests/labels-with-all-legal-characters.yml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: labels-with-all-legal-characters
+  labels:
+    illuminatio-e2e: labels-with-all-legal-characters
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test
+  namespace: labels-with-all-legal-characters
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
```

Comment:
IMHO setting the `revisionHistoryLimit` doesn't make really any sense in the tests since we create the deployment and delete it without changing it.